### ISSUE TITLE
FCL: making transactions with string arguments with colons will result in error

### ISSUE THREAD

**Issue by timongll** [onflow/flow-cli#353]
### Problem

fcl transactions that have string arguments such as 
`flow transactions send ./cadence/transactions/initialize_account.cdc --signer testnet-account --network testnet -f ./flow.json --arg: String:"https://xn--yba991n/"`
will result in the error
`argument not passed in correct format, correct format is: Type:Value, got String:https://xn--yba991n/`
probably due to the colon in the string

**Comment by sideninja** [comment]
Thank you for reporting. This is a valid bug and we will try to solve it as soon as possible, but please feel free and welcome to open a PR yourself.

**Comment by sideninja** [comment]
Happy to let you know this bug has been fixed in the latest version v0.28.0 (soon to be released). 
You will be able to use the command like so:
```
flow transactions send ./cadence/transactions/initialize_account.cdc https://xn--yba991n/ --signer testnet-account --network testnet -f ./flow.json
```
Arguments can now be passed without a type. Stay tuned for version release.
